claim_from_delivery/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# © 2020 Vanmoof BV (<https://www.vanmoof.com>)
+# © 2004-2009 Tiny SPRL (<http://tiny.be>)
```

Comment:
To be precise, I see code changes up until 2015 in odoo/9.0, and Tiny SPRL is currently Odoo SA